demos/select.html

```diff
@@ -142,17 +145,31 @@ <h2 class="mdc-typography--title">Fully-Featured Component</h2>
 
       <section class="example">
         <h2 class="mdc-typography--title">CSS Only</h2>
-        <div class="mdc-select">
-          <select class="mdc-select__surface">
-            <option value="" selected>Pick a food group</option>
-            <option value="grains">Bread, Cereal, Rice, and Pasta</option>
-            <option value="vegetables" disabled>Vegetables</option>
-            <option value="fruit">Fruit</option>
-            <option value="dairy">Milk, Yogurt, and Cheese</option>
-            <option value="meat">Meat, Poultry, Fish, Dry Beans, Eggs, and Nuts</option>
-            <option value="fats">Fats, Oils, and Sweets</option>
-          </select>
-          <div class="mdc-select__bottom-line"></div>
+        <section class="box-demo-wrapper">
+          <div id="demo-css-only-select" class="mdc-select">
+            <select class="mdc-select__surface">
+              <option value="" selected>Pick a food group</option>
+              <optgroup label="First Group">
```

Comment:
I'm not sure we want to use `<optgroup>` in our demos; the spec doesn't really have anything like it AFAIK.